docs/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md

```diff
@@ -197,15 +197,15 @@ The relationship between the default roles deployed by Rancher (i.e. cluster-own
 | project-owner | admin | monitoring-admin | RoleBinding within Project namespace |
 | project-member | edit | monitoring-edit | RoleBinding within Project namespace |
 
-In addition to these default Roles, the following additional Rancher project roles can be applied to members of your Cluster to provide additional access to Monitoring. These Rancher Roles will be tied to ClusterRoles deployed by the Monitoring chart:
+In addition to these default roles, the following additional Rancher project roles can be applied to members of your cluster to provide additional access to monitoring. These Rancher roles will be tied to ClusterRoles deployed by the monitoring chart:
 
 <figcaption>Non-default Rancher Permissions and Corresponding Kubernetes ClusterRoles</figcaption>
 
 | Rancher Role  |  Kubernetes ClusterRole | Available In Rancher From | Available in Monitoring v2 From |
 |--------------------------|-------------------------------|-------|------|
 | View Monitoring* | [monitoring-ui-view](#monitoring-ui-view)    |    2.4.8+    |  9.4.204+ |
 
-\* A User bound to the **View Monitoring** Rancher Role only has permissions to access external Monitoring UIs if provided links to those UIs. In order to access the Monitoring Pane to get those links, the User must be a Project Member of at least one Project.
+\* A User bound to the **View Monitoring** Rancher role only has permissions to access external Monitoring UIs if provided links to those UIs. In order to access the Monitoring Pane to get those links, the User must be a Project Member of at least one Project.
```

Comment:
```suggestion
\* A user bound to the **View Monitoring** Rancher role with read-only permissions won't have access to Grafana links in the monitoring UI. They can still access external Monitoring UIs if provided links to those UIs. In order to access the Monitoring Pane to get those links, the user must be a project member of at least one Project.
```

Although the issue is slated for 2.8, this is known intended behavior and applies to past versions as well. The update for 2.8 is a work-around (?)